action.yaml

```diff
@@ -50,6 +50,7 @@ runs:
         # This /github/home is the internal Docker mount of $RUNNER_TEMP/_github_home
         NODE_PATH: /github/home/node_modules
     - name: Clear commitlint config changes
+      if: hashFiles(inputs.config-file) != ''
       shell: bash
       run: |
         git checkout "${{ inputs.config-file }}"
```

Comment:
Won't this still be a problem if the config-file did not exist and was created on line 43? In that case, the file will exist, but not be under git control?

Reply:
It should only be recreated/updated on L43 if it already exists - see [L39](https://github.com/open-turo/action-pre-commit/blob/main/action.yaml#L39)
![image](https://user-images.githubusercontent.com/54169/161608392-349bec44-9bbb-4293-85e6-2af945015ca1.png)